docs/excel/custom-functions-debugging.md

```diff
@@ -13,43 +13,31 @@ This article discusses debugging only for custom functions that **don't use a [s
 
-This debugging process doesn't work with projects that are created with the **Office Add-in project containing the manifest only** option in the Yeoman generator. The scripts that are referred to later in this article aren't installed with that option. To debug an add-in that is created with this option, see the instructions in one of these articles, as appropriate.
-
-- [Debug add-ins using developer tools in Microsoft Edge (Chromium-based)](../testing/debug-add-ins-using-devtools-edge-chromium.md)
-- [Debug add-ins using developer tools in Internet Explorer](../testing/debug-add-ins-using-f12-tools-ie.md)
-- [Debug Office Add-ins on a Mac](../testing/debug-office-add-ins-on-ipad-and-mac.md)
-
-Use the following anchor links to visit sections of this article that are relevant to your debugging scenario.
-
-On Windows:
-
-- [Excel Desktop and Visual Studio Code (VS Code) debugger](#use-the-vs-code-debugger-for-excel-desktop)
-- [Excel on the web and VS Code debugger](#use-the-vs-code-debugger-for-excel-in-microsoft-edge)
-- [Excel on the web and browser tools](#use-the-browser-developer-tools-to-debug-custom-functions-in-excel-on-the-web)
-- [Command line](#use-the-command-line-tools-to-debug)
-
-On Mac:
-
-- [Excel on the web and browser tools](#use-the-browser-developer-tools-to-debug-custom-functions-in-excel-on-the-web)
-- [Command line](#use-the-command-line-tools-to-debug)
+| **Target platform** | **Visual Studio Code** | **Other IDE** |
+|--------------|-------------|-------------|
+| Excel on Windows | [Use the VS Code debugger for Excel on Windows](#use-the-vs-code-debugger-for-excel-on-windows) | Debugging CF-NSR outside VS Code isn't supported. Debug against Excel on the web. |
+| Excel on the web | Windows development computer: [Use the VS Code debugger for Excel in Microsoft Edge](#use-the-vs-code-debugger-for-excel-in-microsoft-edge)</br>Mac or Windows development computer: [Use VS Code and the browser development tools](#use-the-browser-developer-tools-to-debug-custom-functions-in-excel-on-the-web) | [Use the command line tools](#use-the-command-line-tools-to-debug)|
+| Excel on Mac |  VS Code debugging of CF-NSR isn't supported. Debug against Excel on the web. | [Use the command line tools](#use-the-command-line-tools-to-debug)|
 
 > [!NOTE]
-> For simplicity, this article shows debugging in the context of using Visual Studio Code to edit, run tasks, and in some cases use the debug view. If you are using a different editor or command line tool, see the [command line instructions](#commands-for-building-and-running-your-add-in) at the end of this article.
+> This article primarily shows debugging in the context of using Visual Studio Code to edit, run tasks, and in some cases use the debug view. If you're using a different editor or command line tool, see the [Commands for building and running your add-in](#commands-for-building-and-running-your-add-in) at the end of this article.
```

Comment:
I found this first sentence hard to read. Consider dropping the "in some cases" part to make this list easier to parse.

Also, I think that "see the" should just be "see".
```suggestion
> This article primarily shows debugging in the context of using Visual Studio Code to edit, run tasks, and use the debug view. If you're using a different editor or command line tool, see [Commands for building and running your add-in](#commands-for-building-and-running-your-add-in) at the end of this article.
```